src/gitbackend.c

```diff
@@ -345,3 +345,70 @@ int init_repo(git_repository** repo, char* path, Archive* archive) {
     }
     return attach_archive_to_repo(*repo, archive);
 }
+
+#include <Python.h>
+
+typedef struct CRepository {
```

Comment:
this is a private header so i recreate the struct here so i can get at the repo pointer